networkx/algorithms/planarity.py

```diff
@@ -24,8 +25,29 @@ def check_planarity(G, counterexample=False):
         If the graph is planar `certificate` is a PlanarEmbedding
         otherwise it is a Kuratowski subgraph.
 
+    Examples
+    --------
+
+    This function can be considered as the primary interface for the planar embedding tools since
+    -if the graph is planar- it returns a PlanarEmbedding which can be then used to call other methods of this class.
+    Consider the following example:
+
+    >>> G = nx.Graph([(0,1),(0,2)])
+    >>> isPlanar, P = nx.check_planarity(G)
+    >>> print(isPlanar)
+    True
+
+    Once the PlanarEmbedding object is created using `check_planarity()` as above, the other features of the class can be used safely.
+    Let's see this on action by extending our initial example:
+
+    >>> G = nx.Graph([(0,1),(0,2)])
+    >>> isPlanar, P = nx.check_planarity(G)
```

Comment:
No need to repeat these lines, the scope here encompasses the entire `Examples` section.